src/root/useRestoredReduxState.ts

```diff
@@ -23,6 +23,7 @@ import { restoreUI as restoreSpeciesPageUI } from 'src/content/app/species/state
 import { loadInitialState as loadEntityViewerGeneralState } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralSlice';
 import { restoreGeneViewTranscripts } from 'src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice';
 import { loadBrowserGeneralState } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
+import { loadPreviouslyViewedObjects } from 'src/content/app/genome-browser/state/browser-bookmarks/browserBookmarksSlice';
```

Comment:
Probably want to alias this import in this file. This hook deals with multiple apps at once.

```
import { loadPreviouslyViewedObjects  as loadPreviouslyViewedGenomeBrowserObjects } from ....
```
?

Reply:
✅